src/Pages/GradeSelect/FlightListCard/index.js

```diff
@@ -0,0 +1,230 @@
+import styled from 'styled-components/macro';
+
+function FlightListCard(props) {
+  const arrival = props.list.arrival_datetime.split(' ')[1];
+  const departure = props.list.departure_datetime.split(' ')[1];
+  const price = +props.list.price;
+  const duration = props.list.duration.split(':');
+
+  function handleInput() {
+    props.handleSelected(props.list);
```

Comment:
```suggestion
function FlightListCard({ list }) {
  const [_, arrival] = list.arrival_datetime.split(' ');
  const [__, departure] = list.departure_datetime.split(' ');
  
  const [hour, minute] = list.duration.split(':');

  function handleInput() {
    props.handleSelected(list);
```
- props 구조분해 할당 할 수 있으면 가급적 하는 것이 좋다고 생각합니다!
- 위에 말씀드린 대로 백엔드에서 애초에 데이터를 number로 받아올 수 있다면 숫자로 바꿔주는 로직이 불필요 할 것 같습니다
- duration 역시 구조분해 할당으로 표현해주기 좋은 데이터인 것 같습니다

Reply:
10번에 핸들 함수를 사용하려다 보니 우선은 props가 필요하여 이렇게 정의했습니다